modules/rostests/kmtests/ntos_cc/CcCopyRead_drv.c

```diff
@@ -315,17 +326,23 @@ TestIrpHandler(
         Status = STATUS_SUCCESS;
     }
 
+    // CcCopyRead never sets STATUS_PENDING, especially with wait=TRUE.
+    ASSERT(Status != STATUS_PENDING);
+
+// NB: Same block in IoReadWrite_drv.c, MmMapLockedPagesSpecifyCache_drv.c and NtCreateSection_drv.c.
+// Maybe (not) keep the following STATUS_PENDING checks, as a template.
     if (Status == STATUS_PENDING)
     {
-        IoMarkIrpPending(Irp);
-        IoCompleteRequest(Irp, IO_NO_INCREMENT);
-        Status = STATUS_PENDING;
+//        IoMarkIrpPending(Irp);
+// Remove wrong code added as is in r71445.
     }
     else
     {
         Irp->IoStatus.Status = Status;
         IoCompleteRequest(Irp, IO_NO_INCREMENT);
     }
 
+    trace("TestIrpHandler returns: Status = 0x%08x (%c= STATUS_PENDING)",
```

Comment:
Why are we seeing your debug code in a PR? This belongs in your local working copy to be tested, not in a branch that's under review.

Reply:
Removed the `STATUS_PENDING` part, as related code is now removed too.
`trace` itself is meant as an improvement.